packages/discovery-react-components/src/components/DocumentPreview/__tests__/DocumentPreview.test.tsx

```diff
@@ -246,7 +246,7 @@ describe('DocumentPreview', () => {
         ({ getByText } = render(<DocumentPreview document={errorJsonDoc} />));
       });
 
-      getByText('Cannot preview document');
+      getByText("Can't preview document");
```

Comment:
I wonder how prettier did not catch the double-quotes. Here and on line 313

Reply:
I used single quotes here but my linter has changed it. Just tried again and same thing.

Reply:
Our ESLint rules currently change it to double quotes if there is an escaped character inside the string.